lib/opensusebasetest.pm

```diff
@@ -583,6 +566,26 @@ sub wait_boot {
     $self->{in_wait_boot} = 0;
 }
 
+sub wait_boot_on_local_disk {
```

Comment:
I am not sure about the name. We do not actually wait for the boot to complete as we are waiting for the grub-menu in the end, right? I think we need to find a better name for the method. Can you please also add a doc-string same as for other methods, e.g. as for `wait_boot`

Reply:
I change the name to wait_grub_to_boot_on_local_disk. WDYT?